### PR TITLE
Update notes MySQL version in docker-compose.yml

### PR DESCRIPTION
**Pull request by adzuci**
This is the first step towards getting notes to use mysql57 in production

### CODE REVIEW

**Comment by natabene** [comment]
@fredsmith or @jdmulloy Could you review please when you have a chance?

**Review by jdmulloy — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -247,7 +247,7 @@ services:
     depends_on:
       - devpi
       - elasticsearch
-      - mysql
+      - mysql57
```
Comment (regisb) [inline docker-compose.yml:255]: You will almost certainly have to change `DB_HOST: "edx.devstack.mysql"` to `DB_HOST: "edx.devstack.mysql57"`, no?

Comment (adzuci) [reply docker-compose.yml:255]: Oof, yes, thanks for the catch Regis!  I was rushing and definitely meant to update that part...

**Comment by adzuci** [comment]
Updated the DB_HOST value and rebased the PR.

**Pull request merged**